# Termux/Bionic port: base/Makefile
#
# Bionic bundles no libm/libgcc_s/libstdc++; letting these symlink rules fail is the
# same treatment Julia already gives libssp/libatomic/libgomp/libquadmath.
--- a/base/Makefile
+++ b/base/Makefile
@@ -215,7 +215,7 @@
 # the following excludes: libuv.a, libutf8proc.a
 
 ifneq ($(USE_SYSTEM_LIBM),0)
-$(eval $(call symlink_system_library,LIBM,$(LIBMNAME)))
+$(eval $(call symlink_system_library,LIBM,$(LIBMNAME),,ALLOW_FAILURE))
 else ifneq ($(USE_SYSTEM_OPENLIBM),0)
 $(eval $(call symlink_system_library,OPENLIBM,$(LIBMNAME)))
 endif
@@ -233,12 +233,12 @@
 endif
 endif
 else
-$(eval $(call symlink_system_library,CSL,libgcc_s,1))
+$(eval $(call symlink_system_library,CSL,libgcc_s,1,ALLOW_FAILURE))
 endif
 ifneq (,$(LIBGFORTRAN_VERSION))
 $(eval $(call symlink_system_library,CSL,libgfortran,$(LIBGFORTRAN_VERSION)))
 endif
-$(eval $(call symlink_system_library,CSL,libstdc++,6))
+$(eval $(call symlink_system_library,CSL,libstdc++,6,ALLOW_FAILURE))
 # We allow libssp, libatomic, libgomp and libquadmath to fail as they are not available on all systems
 $(eval $(call symlink_system_library,CSL,libssp,0,ALLOW_FAILURE))
 $(eval $(call symlink_system_library,CSL,libatomic,1,ALLOW_FAILURE))
